web/src/components/navbar/NavBarMobile.tsx

```diff
@@ -15,9 +15,16 @@ interface Props {
   task?: Task;
   hideMiniRoadmap?: boolean;
   showSidebar?: boolean;
+  landing?: boolean;
 }
 
-export const NavBarMobile = ({ scrolled, task, showSidebar, hideMiniRoadmap }: Props): ReactElement => {
+export const NavBarMobile = ({
+  scrolled,
+  task,
+  showSidebar,
+  hideMiniRoadmap,
+  landing,
```

Comment:
I have a slight preference for booleans named clearly to be boolean, ie `isLanding` instead of `landing`